eval/mt_bench/components.py

```diff
@@ -193,13 +193,16 @@ def shutdown_vllm(process: subprocess.Popen, timeout: int = 20):
         os.path.join(models_path_prefix, best_model),
         os.path.join(models_path_prefix, "candidate_model"),
     )
-    best_model = f"{models_path_prefix}/candidate_model"
+    best_model_renamed = os.path.join(models_path_prefix, "candidate_model")
+    best_model_output = f"Candidate model: {best_model} located at {best_model_renamed}"
```

Comment:
Why does it have to be a message? Later on we use this to print the best model.

Reply:
Job logs:

```
INFO 2024-10-18 11:29:26,271 __main__:2433: Job completed successfully.
INFO 2024-10-18 11:29:26,792 __main__:956: Best model: /data/model/output/phase_2/hf_format/samples_320
INFO 2024-10-18 11:29:26,792 __main__:3140: Running final evaluation.
```

Pod log:

```
{
"best_model": "/data/model/output/phase_2/hf_format/samples_320",
"best_score": 8.5
}
```

Later on for final eval, we know the model has been renamed to `candidate_model` so everything is fine.

Reply:
Because the script uses the Pod logs to print the best model, the content of `best_model` should not be a sentence.

Reply:
true, but the pipeline uses the output to find the model for final eval - for now, I've hard-coded that value as `../hf_format/candidate_model` and removed the updated output here.